### DISCUSSION TITLE
how to tune 144.174MHz USB?

### DISCUSSION THREAD

**Discussion by jurajg** [egzumer/uv-k5-firmware-custom#412] [Q&A]
Hello, I have UV K5(8). Stock firmware has lowest step 2.5kHz. If I flashed egzumer, am i able to tune exactly to  **144.174MHz**? Is it possible to tune by **1kHz** steps or there is an hardware limitation?
thanks

**Comment by gartnl** [comment]
Adust 'step' in menu, as low as 0.1 kHz.
